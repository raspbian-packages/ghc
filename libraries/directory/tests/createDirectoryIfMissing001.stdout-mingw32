()
CreateDirectory "createDirectoryIfMissing001.d\\a": does not exist (The system cannot find the path specified.)
()
()
()
()
CreateDirectory "createDirectoryIfMissing001.d": already exists (Cannot create a file when that file already exists.)
CreateDirectory "createDirectoryIfMissing001.d": already exists (Cannot create a file when that file already exists.)
