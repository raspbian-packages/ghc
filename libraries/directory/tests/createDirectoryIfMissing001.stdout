()
createDirectoryIfMissing001.d/a: createDirectory: does not exist (No such file or directory)
()
()
()
()
createDirectoryIfMissing001.d: createDirectory: already exists (File exists)
createDirectoryIfMissing001.d/a: createDirectory: inappropriate type (Not a directory)
